Sources/OTel/OTel.docc/Metrics/metrics.md

```diff
@@ -0,0 +1,106 @@
+# Metrics
+
+@Metadata {
+    @TitleHeading("Signal")
+    @PageImage(purpose: card, source: "metrics-card", alt: "An illustration of diagrams.")
+    @PageColor(purple)
+    @Available("Swift OTel", introduced: "1.0")
+}
+
+OpenTelemetry metrics implemented on top of Swift Metrics.
```

Comment:
I think we should be deliberate about the terms we use here, and I would suggest we decouple OTLP from the bigger claim of _OpenTelemetry/OTel Metrics_.

The current line that this package is walking is to be a backend for the Swift observability API packages—Swift Distributed Tracing, Swift Metrics, and Swift Log—and to emit measurements using OTLP suitable for an OTel Collector.

As we've discussed before, Swift Distributed Tracing and OTel Tracing offer very similar functionality and API surface for recording traces, but the same is not true when comparing Swift Metrics and OTel Metrics.

In my mind there are two audiences here that we could consider:
1. OTel-familiar, Swift-curious: expect to find an OTel library that offers the usual OTel API/SDK from the spec.
2. Swift-familiar, OTel-curious: expect to bootstrap the Swift Metrics system with a backend that exports to an OTel collector.

We are currently firmly in camp (2), to the point that I think the headline for this should change. 

```suggestion
OTLP backend for Swift Metrics.
```

Reply:
That's good points Si, the headline sounds good :)